pkg/build/prometheus/metrics.go

```diff
@@ -0,0 +1,172 @@
+package prometheus
+
+import (
+	kselector "k8s.io/apimachinery/pkg/labels"
+	"strings"
+
+	"github.com/golang/glog"
+	buildapi "github.com/openshift/origin/pkg/build/apis/build"
+	internalversion "github.com/openshift/origin/pkg/build/generated/listers/build/internalversion"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	separator                = "_"
+	BuildSubsystem           = "openshift_build"
```

Comment:
I use them in the extended tests.  Also saw that convention used in https://github.com/openshift/origin/pull/14796/files#diff-5ed808e701b743f946088cc61a6ecf81R12